docs/docs/tutorials/helidon-integration.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 29
+---
+
+# Helidon Integration
+
+[Helidon](https://helidon.io/) provides a LangChain4j integration module that simplifies to build AI-driven applications while leveraging Helidon’s programming model and style.
+
+You can find the detailed explanation and usage of LangChain4j integration feature [here](https://helidon.io/docs/latest/se/integrations/langchain4j/langchain4j).
+
+## Supported versions
+
+Helidon's LangChain4j integration requires Java 21 and Helidon 4.2.
```

Comment:
Please provide a link to Helidon 4.2 announcement: https://medium.com/helidon/helidon-4-2-released-fa94afe4945a

Reply:
I have provided link to main Helidon 4.2 documentation above.